packages/console/src/pages/SignInExperience/index.tsx

```diff
@@ -90,14 +92,18 @@ const SignInExperience = () => {
     await saveData();
   });
 
-  if ((!settings && !settingsError) || (!data && !error)) {
+  if ((!settings && !settingsError) || (!data && !error) || isLoadingLanguages) {
     return <Skeleton />;
   }
 
   if (!settings && settingsError) {
     return <div>{settingsError.body?.message ?? settingsError.message}</div>;
   }
 
+  if (languageError) {
+    return <div>{languageError.body?.message ?? languageError.message}</div>;
+  }
+
```

Comment:
I know this is kinda unrelated to your PR, but this div seems unstyled. Need to either popup the error message in a toast, or display the error page instead. Can you confirm this and create a task if necessary?

Reply:
[LOG-4326](https://linear.app/silverhand/issue/LOG-4326/handle-un-styled-error-page-state-in-sie-page)